cmd/cloudFoundryDeleteService_test.go

```diff
@@ -0,0 +1,63 @@
+package cmd
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestCloudFoundryDeleteService2(t *testing.T) {
+	execRunner := execMockRunner{}
+	/*
```

Comment:
This should be deleted (if not needed)

Reply:
I suppose you ask to delete this due to the naming "TestCloudFoundryDeleteService2", which causes confusion. Changed the naming accordingly so it is named "TestCloudFoundryDeleteService" now. Hope that should resolve this issue.